### DISCUSSION TITLE
Append multiple images from the same folder

### DISCUSSION THREAD

**Discussion by Chris9178** [ImageMagick/ImageMagick#6903] [Digital Image Processing] [answered]
Hello!

I was wondering how can I execute a command in order to automaticly get all the images within a same folder and then -append them into one image?

Thanks a lot for help!

**Comment by fmw42** [comment]
What is your Imagemagick version and platform/OS?

**Reply by Chris9178** [reply]
Hey, thanks for answer!

I'm on Windows 10 with this vers:  7.1.1-21-Q16-HDRI-x64-dll.exe

**Reply by fmw42** [reply, marked as answer]
Try

```
cd folder_with_images
magick * -append result.suffix
```

**Reply by Chris9178** [reply]
Amazing, it's perfect!
Thanks a lot!!!

**Comment by snibgo** [comment]
I suggest inserting `+repage` after the append, to set the canvas size to the size of the result.
